conda-forge.yml

```
@@ -1,11 +1,15 @@
-build_platform: {osx_arm64: osx_64}
+azure:
+  store_build_artifacts: true
+build_platform:
+  osx_arm64: osx_64
```

This would enable cross-compilation (with a re-render after)

```suggestion
build_platform:
  linux_aarch64: linux_64
  linux_ppc64le: linux_64
  osx_arm64: osx_64
```